### DISCUSSION TITLE
No Chromecast?

### DISCUSSION THREAD

**Discussion by Zuko** [jellyfin/jellyfin-desktop#157] [General]
Why Chromecast isn't available? I can only cast from phone/tablet...

![image](https://user-images.githubusercontent.com/521035/138514687-021e1436-ce95-4e3a-baff-c7242a96ac50.png)

**Comment by iwalton3** [comment]
I believe Chromecast support requires some proprietary Google Chrome components.

**Reply by Zuko** [reply]
I don't think so _(but maybe i'm wrong)_
VLC has it: https://github.com/videolan/vlc/tree/master/modules/stream_out/chromecast

**Comment by dmgolembiowski** [comment]
> I believe Chromecast support requires some proprietary Google Chrome components.

From my recent experiences with Chromecast, it seems full-blown https is needed. 🥲

**Reply by Phlogi** [reply]
Where is this documented?
